### PR TITLE
Modals: Display in source-window

### PR DESCRIPTION
**Pull request by BurningTreeC**
this makes modals display within the window where they got opened, with the parameter `rootwindow` that, if `yes` or `true`, shows the modal always in the root TW window (`<$action-sendmessage $message="tm-modal" $param="mymodal" rootwindow="yes|true"/>`)

### CODE REVIEW

File: core/modules/utils/dom/modal.js [hunk 1]
```diff
@@ -26,8 +26,12 @@ Display a modal dialogue
 Options include:
 	downloadLink: Text of a big download link to include
 */
-Modal.prototype.display = function(title,options) {
+Modal.prototype.display = function(event,options) {
```
Comment (Jermolene) [inline core/modules/utils/dom/modal.js]: We shouldn't change the type of an existing parameter. Perhaps we can use `options.event` instead?

Comment (BurningTreeC) [reply core/modules/utils/dom/modal.js]: @Jermolene , done!

**Comment by BurningTreeC** [comment]
I'll add a note about the rootwindow attribute in the docs

**Comment by Jermolene** [comment]
Thanks @BurningTreeC ping me when you're ready

**Comment by BurningTreeC** [comment]
Hi @Jermolene , I've added docs about the new `rootwindow` attribute

**Comment by Jermolene** [comment]
Thanks @BurningTreeC

**Pull request merged**